spec/requests/posts_spec.rb

```diff
@@ -1,7 +1,33 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+  describe 'GET posts#index' do
```

Comment:
- [OPTIONAL] As you did for the users_spec kindly make sure your posts Spec file includes tests that ensure the correct view template is rendered. Example : `expect(response).to render_template(:show)`